infra/core/database/cosmos-db/account.bicep

```diff
@@ -56,7 +56,14 @@ resource account 'Microsoft.DocumentDB/databaseAccounts@2024-05-15' = {
               name: 'EnableNoSQLVectorSearch'
             }
           ]
-        : []
+        : [],
+      (enableNoSQLFullTextSearch)
```

Comment:
This parameter isn't defined anywhere and throws an error during bicep deployment. Need to add parameter definition at the top of this file.

Add this below line 15
```
@description('Enables NoSQL full text search for this account. Defaults to false.')
param enableNoSQLFullTextSearch bool = false
```

And we need to pass a value in from the `infra/app/database.bicep` file. Add the parameter below so starting on line 140 it should look like this
```
module cosmosDbAccount '../core/database/cosmos-db/nosql/account.bicep' = {
  name: 'cosmos-db-account'
  params: {
    name: accountName
    location: location
    tags: tags
    enableServerless: true
    enableVectorSearch: true
    enableNoSQLFullTextSearch: true // This is the new parameter we just defined
    disableKeyBasedAuth: true
  }
}
```

Reply:
Added in the necessary places.